src/context/TagsContext.js

```diff
@@ -25,17 +26,31 @@ export function TagsProvider(props) {
   // EVENT HANDLERS
   const loadTags = async () => {
     const tags = await api.getTags();
```

Comment:
I'd suggest renaming this to `result` or `response` so that we don't end up with `tags.tags`.